### ISSUE TITLE
truffle-config.js created by "truffle init" mentions deprecated "truffle-hdwallet-provider"

### ISSUE THREAD

**Issue by fodisi** [trufflesuite/truffle#2854]
- [ ] I've asked for help in the [Truffle Gitter](http://gitter.im/Consensys/truffle) before filing this issue.

---------------------------

## Issue

When using the command `truffle init`, the default `truffle-config.js` file is mentioning the deprecated "truffle-hdwallet-provider" package. It'd be good replace it by "@truffle/hdwallet-provider".

## Steps to Reproduce

In a new directory, run the command `truffle init`

## Expected Behavior

`truffle-config.js` to mention `@truffle/hdwallet-provider` as bellow:

```
/**
...
 * To deploy via Infura you'll need a wallet provider (like @truffle/hdwallet-provider)
...
 */

// const HDWalletProvider = require('@truffle/hdwallet-provider');
```

## Actual Results
```
/**
...
 * To deploy via Infura you'll need a wallet provider (like truffle-hdwallet-provider)
...
 */

// const HDWalletProvider = require('truffle-hdwallet-provider');
```

## Environment

* Operating System: 
* Ethereum client:
* Truffle version (`truffle version`): v5.1.14
* node version (`node --version`):
* npm version (`npm --version`):

**Comment by fodisi** [comment]
After checking the code for `truffle init`, I noticed it actually refers to [https://github.com/truffle-box/bare-box](https://github.com/truffle-box/bare-box). So, I believe the issue should be fixed in this truffle box. Is that correct?

_PS: I'm considering the the above stands correct, so I just submitted this [PR22](https://github.com/truffle-box/bare-box/pull/22) to the box's repo to get it fixed. In case this is not the place to get it fixed, please let me know and I'll gladly investigate further to try to provide a proper fix. Any advise is welcomed. Thanks. :smiley:_

**Comment by CruzMolina** [comment]
Thanks again @fodisi !

Resolved via #truffle-box/bare-box#22.
